### ISSUE TITLE
question about the accuracy

### ISSUE THREAD

**Issue by flexibility2** [dragen1860/LearningToCompare-Pytorch#9]
Hi, thanks for your source code, however when I run your source code, the accuracy is:
1)  51.9%  for the 5-way-1-shot
2)  65.3%  for the 5-way-5-shot 
the accuracy is far away from the what you have published (they should be 57.2% and 71.07% respectively in the form you posted )
so I am wondering if you could explain this problem, thanks a lot!

**Comment by dragen1860** [comment]
It's the matter of hyper-parameters. currently I  dont focus on this area so I cant give you more hits. If you are doing research, that's cool for you to improve it.

**Comment by flexibility2** [comment]
Sorry to bother you again, **but I just want to confirm with you whether you have achieved those results that you posted in your README,** which are 57.2% and 71.07% for 1-shot and 5-shot.

Because after a few days attempt based on your source code, it seems the results don't improve compared to the author's code which didn't use the deep model. 

I would be very appreciated if you could replay.
@dragen1860
